_browser/blockstack_storage.md

```diff
@@ -630,40 +577,25 @@ process URL paths that contain the `.` (dot) character for example,
      documentation](http://blockstack.github.io/blockstack.js/#getfile) for
      details.
 
-5. Add the following block to `fetchData()` right after the call to `lookupProfile(username)... catch((error)=>{..}` block:
+5. Add the following block to `fetchData()` right after the call to `setUsername(username)` block:
 
     ```javascript
     const options = { username: username, decrypt: false }
-    userSession.getFile('statuses.json', options)
-      .then((file) => {
-        var statuses = JSON.parse(file || '[]')
-        this.setState({
-          statusIndex: statuses.length,
-          statuses: statuses
-        })
-      })
-      .catch((error) => {
-        console.log('could not fetch statuses')
-      })
-      .finally(() => {
-        this.setState({ isLoading: false })
-      })
+    const file = await userSession.getFile('statuses.json', options)
+    const _statuses = JSON.parse(file || '[]')
+    setStatusIndex(_statuses.length);
+    setStatuses(_statuses);
+    setLoading(false);
     ```
 
     This fetches the user statuses.
 
     Finally, you must conditionally render the logout button, status input textbox, and submit button so they don't show up when viewing another user's profile.
 
-6. Replace the `render()` method with the following:
+6. Replace the returned JSX in the `Profile` component with the following:
```

Comment:
The first time we reference "JSX". I'd rather avoid that

Reply:
I would push back and say that just because it's not mentioned before doesn't mean we should not add it. The tutorial assumes some level of react experience.

Reply:
What's the win of adding "JSX" here as opposed to just referring to the `return` method? I don't see any upside but a potential confusion. 

Referring to JSX is fine by itself, I'm not saying we can't assume them to know about it. But, if we want to use the term, let's use it also in one of the first steps where we replace the render method with the return method?

Reply:
@aulneau there is some confusion because Alex was using the current version of the app generator, which uses a class component. But my updated PR for the generator uses functional components. So there is no `render` method.

Reply:
@hstove yeah def! I think it could be fine to rephrase it such that it references updating what is returned. I also think using JSX across the board is more specific and correct.